control-plane/msp-operator/src/main.rs

```diff
@@ -797,6 +831,13 @@ async fn reconcile(
             return msp.pool_check().await;
         }
 
+        Some(MayastorPoolStatus {
```

Comment:
We can move this to the above lines, where we check if the pool is Online. Something like:

```rust
match { 
Some(A) | Some(B) => { }
}
```